src/itest/java/uk/gov/companieshouse/company/metrics/config/CucumberSpringConfiguration.java

```diff
@@ -1,13 +1,11 @@
-package uk.gov.companieshouse.company.metrics;
+package uk.gov.companieshouse.company.metrics.config;
 
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.TestPropertySource;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @DirtiesContext
 @ActiveProfiles({"test"})
-public abstract class AbstractIntegrationTest {
-
+public class CucumberSpringConfiguration extends AbstractMongoConfig {
```

Comment:
Probably better to leave name as AbstractIntegrationTest, as its a baseItest instead of config class